as/cNFT/assembly/models/persistent_tokens.ts

```diff
@@ -1,10 +1,13 @@
-import {
-    PersistentSet,
-    PersistentUnorderedMap,
-    u128,
-} from 'near-sdk-as'
+import { PersistentSet, PersistentUnorderedMap, u128 } from 'near-sdk-as'
 import { AccountId, TokenId } from '../types'
-import { Token } from './token'
+
+@nearBindgen
+export class Token {
+    id: string
+    owner_id: string
+    creator: string
+    prev_owner: string
```

Comment:
Shouldn't `creator` be called `creator_id` and `prev_owner` called `prev_owner`. 
To keep it the consistent with `owner_id`

Reply:
Yes, better :)